analyzers/src/SonarAnalyzer.VisualBasic/Extensions/ExpressionSyntaxExtensions.Roslyn.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.CodeAnalysis.VisualBasic.Extensions;
+
+[ExcludeFromCodeCoverage]
+internal static class ExpressionSyntaxExtensions
+{
+    // Copy of
+    // https://github.com/dotnet/roslyn/blob/5a1cc5f83e4baba57f0355a685a5d1f487bfac66/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Extensions/ExpressionSyntaxExtensions.vb#L362
+    [ExcludeFromCodeCoverage]
```

Comment:
This method is excluded from coverage explicitely, at method level.
However, the methods below, such as `IsOnlyWrittenTo`, are not.
Is there a specific reason for that?

None of the methods in `Microsoft.CodeAnalysis.Shared.Extensions.SyntaxNodeExtensions` are directly marked as `[ExcludeFromCodeCoverage]`.